### ISSUE TITLE
[BUG] extension stops working few hours after the installation

### ISSUE THREAD

**Issue by mr-d-r** [Andrews54757/FastStream#172]
**Describe the bug**
FastStream lastest version, even downloaded from github, stops working few hours after installation into the lastest chrome with the following error:

![image](https://github.com/user-attachments/assets/affdf8f9-e6f6-4670-a516-895bc7a997d0)

previous FastStream worked fine.

**Version**
1.3.11

**Platform**
win10 chrome 126.0.6478.127 (Official Build) (64-bit) (cohort: Stable)

**Steps to Reproduce**
install it manually, download few videos from YT, wait for 1-2 hours

thank you in advance

**Comment by Andrews54757** [comment]
I can't reproduce this issue. Has anyone else had the same issue?

**Comment by mr-d-r** [comment]
the bug is stable.
I've already reinstalled the extension three times with the same result.

I can turn on debug and send you some logs
if it helps to figure out the root cause.
Just send me instructions how to do it.

![image](https://github.com/user-attachments/assets/35397789-edcc-44d1-b75c-072afe073b53)

![image](https://github.com/user-attachments/assets/0f3f031e-a075-4e95-b5c3-5694d552ea09)

![image](https://github.com/user-attachments/assets/4fb42e59-7380-471f-8976-b28b5da38cf9)

![image](https://github.com/user-attachments/assets/1d7d0b9f-27ea-4888-8bd4-92c3a5d56b8c)

![image](https://github.com/user-attachments/assets/958332a4-e768-4684-952f-f8783f2c8d2a)

**Comment by mr-d-r** [comment]
after chrome restart the extension disappears 


![image](https://github.com/user-attachments/assets/83c7aeed-cdbb-4478-b9e0-1a396e9d5ba2)

**Comment by Andrews54757** [comment]
Did you follow the instructions in the README? You have to turn on debug mode otherwise Chrome will auto delete the extension.

**Comment by Andrews54757** [comment]
Also, you must not delete the folder where faststream is located if you install it manually. I recommend moving it out of your downloads folder into a permanent location before installing it. Also check to make sure your computer's anti virus is not automatically removing it.

**Comment by mr-d-r** [comment]
> Did you follow the instructions in the README? 

 - Go to chrome://extensions
 - Turn on developer mode
 - Drag and drop the chrome directory of this repository
yes. I've followed them.


> You have to turn on debug mode otherwise Chrome will auto delete the extension.

you mean developer mode?
sure


> you must not delete the folder where faststream is located

here is my point of failure. I've made manual install of chrome extension for the first time in my life.
and i was sure chrome will copy it to default folder  :-(
SOLVED

thank you for fast replies!
